src/containers/platform/PlatformGovernmentPage.js

```diff
@@ -0,0 +1,165 @@
+/*
+ * @flow
+ */
+
+import React from 'react';
+
+import styled from 'styled-components';
+import { Colors, Typography } from 'lattice-ui-kit';
+
+import ComplianceAndSecuritySection from './ComplianceAndSecuritySection';
+import CorePlatformSection from './CorePlatformSection';
+
+import FooterSection from '../footer/FooterSection';
+import { BarsCircleIcon, LockCircleIcon, QuestionCircleIcon } from '../../assets/svg/circle-icons';
+import { PlatformLayersGraphic } from '../../assets/svg/graphics';
+import {
+  ContentGrid,
+  GraphicShot,
+  Header,
+  InfoTile,
+  PageSection,
+  ScrollToTop,
+  SectionContent,
+  SolidButton,
+  SubHeader,
+} from '../../components';
+import { openBeacon } from '../../utils/Utils';
+
+const { NEUTRAL } = Colors;
+
+// TODO: kill InfoTile, replace with grid-based Tile
+const Tile = styled.div`
+  display: grid;
+  grid-gap: ${({ gap }) => (gap ? `${gap}px` : '24px')};
+`;
+
+const PlatformGovernmentPage = () => (
+  <>
+    <ScrollToTop />
+    <PageSection bgColor={NEUTRAL.N50}>
+      <SectionContent margin="20vh 0" maxWidth={{ sm: 450, lg: 700 }}>
+        <Header>The Modern OpenLattice Platform</Header>
+        <SubHeader>Finally, a safe and easy way to get actionable insights to public servants.</SubHeader>
+      </SectionContent>
+    </PageSection>
+    <PageSection>
+      <SectionContent>
+        <SectionContent margin="24px 0">
+          <Typography variant="h2">
+            Feed Data In
+          </Typography>
+          <Typography variant="body1">
+            Pull together data from any systems you’d like.
+          </Typography>
+        </SectionContent>
+        <SectionContent margin="24px 0">
+          <Typography variant="h2">
+            Build On Top
+          </Typography>
+          <Typography variant="body1">
+            Use open APIs to connect any tools you’d like.
+          </Typography>
+        </SectionContent>
+        <SectionContent margin="24px 0">
+          <Typography variant="h2">
+            Get Data Out
+          </Typography>
+          <Typography variant="body1">
+            Connect visualization and analysis tools of your choice.
+          </Typography>
+        </SectionContent>
+        <SectionContent margin="24px 0">
+          <Typography variant="h2">
+            Share Data
+          </Typography>
+          <Typography variant="body1">
+            Ensure those who need certain data can access it.
+          </Typography>
+        </SectionContent>
+        <SectionContent margin="24px 0">
+          <Typography variant="h2">
+            While Fully Compliant
+          </Typography>
+          <Typography variant="body1">
+            Meet pretty much any compliance out there.
+          </Typography>
+        </SectionContent>
```

Comment:
no change requested, but I think this block could easily be implemented as a list component and avoid all the repetition.

Reply:
Moved this into its own component.